cypress/integration/message-renderer/adaptiveCard.spec.ts

```diff
@@ -0,0 +1,135 @@
+describe('Message Renderer Theme', () => {
```

Comment:
the test scenario name is incorrect (copy/paste?)

Reply:
yeah, sorry. two places forgot to change it.